package.json

```diff
@@ -23,13 +22,12 @@
     ]
   },
   "scripts": {
-    "prebuild": "npm run clean",
+    "prebuild": "node -e \"var s=require('shelljs'),d=['generators','es5-check','es5-test'];s.rm('-rf',d);s.mkdir('-p',d)\"",
```

Comment:
The generator could detect if those files exist and ask the user whether they'd like the files removed because they are no longer necessary. Or something.